docs/docs-source/docs/modules/get-started/pages/setup-k8s-cluster.adoc

```diff
@@ -0,0 +1,77 @@
+* Go to the Github release page for your chosen Cloudflow version and download the bootstrap script. It's in the list of assets. For example, this is the https://github.com/lightbend/cloudflow/releases/tag/v1.3.3[page] for Cloudflow 1.3.3.
+* Run the bootstrap script to install Cloudflow.
+
+This will take you through the process of installing Cloudflow in the cluster. In course of this it will ask a few questions regarding the storage class to be installed. The following should be the response to these:
+
+```
+Select a storage class for workloads requiring persistent volumes with access mode 'ReadWriteMany'.
+Examples of these are Spark and Flink checkpointing and savepointing.
+
+   NAME                 PROVISIONER                             SUPPORTS RWM?           DEFAULT?
+1. standard             kubernetes.io/gce-pd                    Unknown                 -
+2. nfs                  cluster.local/nfs-server-provisioner    Unknown                 -
+> 2
+
+Select a storage class for workloads requiring persistent volumes with access mode 'ReadWriteOnce'.
+Examples of these are Kafka, Zookeeper, and Prometheus.
+
+   NAME                 PROVISIONER                             SUPPORTS RWO?            DEFAULT?
+1. standard             kubernetes.io/gce-pd                    Verified                 -
+2. nfs                  cluster.local/nfs-server-provisioner    Unknown                  -
+> 1
+```
+
+It then takes you through the installation of the Cloudflow operator, the Strimzi Kafka operator, the Spark operator and the Flink operator. If things go ok, you will see something like the following on your console:
+
+```
++------------------------------------------------------------------------------------+
+|                      Installation of Cloudflow has completed                       |
++------------------------------------------------------------------------------------+
+
+NAME                                                         READY   STATUS              RESTARTS   AGE
+cloudflow-flink-operator-8588dbd8f4-gsjnm                    1/1     Running             0          70s
+cloudflow-operator-57f47676f7-svbvj                          0/1     ContainerCreating   0          3s
+cloudflow-sparkoperator-69669fdd54-88brw                     0/1     ContainerCreating   0          35s
+strimzi-cluster-operator-7ff64d4b7-c9r2n                     1/1     Running             0          51s
+```
+
+The complete installation takes a bit of a time, and you can follow the progress using `kubectl` command. All operators are installed in the namespace `cloudflow`. When everything is installed, you will see the following:
```

Comment:
```suggestion
The complete installation takes a few minutes. 
All operators are installed in the namespace `cloudflow`. 
You can follow the progress of the installation using the following `kubectl` command:
``bash
$ kubectl get pods -n cloudflow
``
When everything is installed, you will see the following output:
```

Reply:
the suggestion syntax prevents me from using the triple quotes in the snippet. I made them double-quotes for the example but you'll need to update it before merging.

Reply:
Fixed.